projects/train/configs/similarity/cbc.yaml

```diff
@@ -20,7 +20,6 @@ model:
   class_path: train.models.SimilarityModel
   init_args:
     outdir: ${oc.env:AMPLFI_OUTDIR}
-    checkpoint: null
```

Comment:
Is there a reason you removed this? In principle it shouldn't break anything, but I may be wrong.